test/spec/fit-bounds.spec.js

```diff
@@ -83,3 +83,32 @@ test('WebMercatorViewport.fitBounds', (t) => {
   }
   t.end();
 });
+
+test('fitBounds#degenerate', (t) => {
+  const OPTIONS = {
+    height: 100,
+    width: 100,
+    bearing: 0,
+    pitch: 0,
+    zoom: 2
+  };
+
+  const viewport = new WebMercatorViewport(OPTIONS);
+  t.doesNotThrow(
```

Comment:
Worth adding a test for the new `maxZoom`/`minZoom` options too?

Reply:
> Worth adding a test for the new maxZoom/minZoom options too?

Agreed. Would you consider doing the honors? :)

Reply:
Done! #100